kubernetes/configs/dev/backend_for_frontend.sh

```diff
@@ -18,3 +18,6 @@ BFF_RATE_LIMITER_MAX_REQUESTS="10"
 BFF_RATE_LIMITER_BURST_PERCENTAGE="1.0"
 BFF_RATE_LIMITER_RATE_INTERVAL="1.second"
 BFF_RATE_LIMITER_TIMEOUT="300.milliseconds"
+
+SUPPORT_SAML_CALLBACK_URL=""
+SUPPORT_SAML_CALLBACK_ERROR_URL=""
```

Comment:
@beetlecrunch can you provide these urls?

Reply:
```suggestion
SUPPORT_SAML_CALLBACK_URL="https://selfcare.dev.interop.pagopa.it/ui/it/assistenza/scelta-ente#id="
SUPPORT_SAML_CALLBACK_ERROR_URL="https://selfcare.dev.interop.pagopa.it/ui/it/assistenza/errore"
```